### DISCUSSION TITLE
Characters in annotations don't display properly

### DISCUSSION THREAD

**Discussion by Ningning1046** [sumatrapdfreader/sumatrapdf#2860] [Q&A]
Hi guys, the characters in annotations don't display properly. I guess it's because the characters aren't in the font library, but I'm not sure about it and I don't know how to add characters in the font library. May I ask that is there any way can make the character display properly? I'd appreciate it if someone would tell me 🙏🙏

![image](https://user-images.githubusercontent.com/72305399/183408219-0c11f0ac-e805-4942-aa5f-45f582c7aed6.png)

**Comment by GitHubRulesOK** [comment]
Annotations in PDF historically have limited support for Unicode see the emojis do not embed well into free text and there can be missing characters in many eastern languages

![image](https://user-images.githubusercontent.com/12754913/183413689-fb482954-3d90-494c-9be7-dc26c39f7e3b.png)
![image](https://user-images.githubusercontent.com/12754913/183416448-6adf3d93-3b98-48fe-ad9d-cebdf86e2981.png)
